docs/lib/datastore/fragments/ios/other-methods.md

```diff
@@ -0,0 +1,49 @@
+## Clear
+
+To clear local data from DataStore, use the `clear` method:
+
+```swift
+Amplify.DataStore.clear { (result) in
+    switch(result){
+    case .success:
+        print("DataStore cleared")
+    case .failure(let error):
+        print("Error clearing DataStore:  \(error)")
+    }
+}
+```
```

Comment:
Removing enclosing parentheses and adding ` ` after keyword `switch`:

```suggestion
```swift
Amplify.DataStore.clear { result in
    switch result {
    case .success:
        print("DataStore cleared")
    case .failure(let error):
        print("Error clearing DataStore:  \(error)")
    }
}
```
```